client/src/utils/mockData.js

```diff
@@ -310,3 +310,10 @@ export const MOCK_TRACKER_DATA = [
     appliedDate: null,
   },
 ];
+
+export const MOCK_SIGNED_IN_USER = {
+  email: 'boba@email.com',
+  firstName: 'Harrison',
+  lastName: 'Ford',
+  password: 'password',
```

Comment:
I know this is mocked but idt we should have the user's password at all in the frontend. For the purposes of this ticket, all that it would do is make it so that the stars representing the password in preview mode have an arbitrary length, and the user's actual password is not shown when they go into edit mode.